packages/icons/src/index.js

```diff
@@ -49,8 +49,10 @@ export { default as color } from './library/color';
 export { default as column } from './library/column';
 export { default as columns } from './library/columns';
 export { default as comment } from './library/comment';
-export { default as commentAuthor } from './library/comment-author';
-export { default as postCommentContent } from './library/post-comment-content';
```

Comment:
`commentAuthor` and `postCommentContent` are already published to npm:
https://unpkg.com/@wordpress/icons@6.0.1/build/index.js

We will have to call it a breaking change in the CHANGELOG of the package.

Reply:
I changed my mind. Given that those new icons were added specifically for core blocks I doubt they are used anywhere else. They weren't included in the CHANGELOG before or exposed in the documentation. Let's list new icons in the CHANGELOG as new features as proposed in https://github.com/WordPress/gutenberg/pull/36171/commits/de40719e7e22810143749f0aa4cb19dd0d745595.